infrastructure/kube/keep-test/geth-node/eth-goerli-internal-service.yaml

```
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: geth-goerli
+  labels:
+    app: geth
+    type: goerli
```

We could label `app` and `network`, I'm not sure we need `type`.
```suggestion
    app: geth
    network: goerli
```

This would be consistent with how we labeled clients, e.g.:
https://github.com/keep-network/keep-core/blob/8abd488a746344d48121ea0ed63ff61f2dc5175e/infrastructure/kube/keep-test/keep-client-3-statefulset.yaml#L5-L11